common/EndpointEncoding.go

```diff
@@ -65,12 +64,11 @@ func (e *Encoding) ToDynamoDB() map[string]ddbtypes.AttributeValue {
 GenerateNumericId generates a (theoretically) unique numeric ID
 */
 func GenerateNumericId() int32 {
-	content, err := GenerateUuidBytes()
```

Comment:
This should at least not use number up to 99999 like the current code as these are already in use.